tutorlimesurvey/plugin.py

```diff
@@ -20,15 +20,17 @@
         # Each new setting is a pair: (setting_name, default_value).
         # Prefix your setting names with 'LIMESURVEY_'.
         ("LIMESURVEY_VERSION", __version__),
-        ("LIMESURVEY_DOCKER_IMAGE", "docker.io/acspri/limesurvey:6.1.0"),
+        ("LIMESURVEY_DOCKER_IMAGE", "docker.io/edunext/limesurvey:15.0.0-tmp.0.1"),
```

Comment:
Can you use `DOCKER_IMAGE_LIMESURVEY` ?

Reply:
I thought about it but the in-line comments (from the cookie cutter) say `# Prefix your setting names with 'LIMESURVEY_'`